### PR TITLE
Resolve failure in introspection following a console edit

### PR DESCRIPTION
**Pull request by rjeberhard**
We discovered that when a domain is restarted following an edit of configuration using the console, that the introspector script would sometimes fail reading beans that were no longer present.

This change wraps those bean lookups in a function that handles the exceptions better.

### CODE REVIEW

**Comment by rjeberhard** [comment]
Ha, ha... It wouldn't show in the diff @markxnelson , but there was already a function there with that naming pattern.  I'm more than happy to change it.  It's "WA" for "work around" ;)

File: operator/src/main/resources/scripts/introspectDomain.py [hunk 1]
```diff
@@ -325,7 +332,7 @@ def validateAdminServer(self):
       addError("The admin server '" + adminServerName + "' does not exist.")
       return
     cluster = adminServer.getCluster()
```
Comment (tbarnes-us) [inline operator/src/main/resources/scripts/introspectDomain.py]: Do we need a 'WA(' method for 'getCluster()' too?

Comment (rjeberhard) [reply operator/src/main/resources/scripts/introspectDomain.py]: I wondered about that...  That method is all over and we've not seen this use case fail on getCluster().

Comment (markxnelson) [reply operator/src/main/resources/scripts/introspectDomain.py]: ah, i was trying to work out who "wrapper" became "WA"

Comment (markxnelson) [reply operator/src/main/resources/scripts/introspectDomain.py]: what about something like getClusterOrNone
or a more generic getOrElse( thingYouWant, None )

Comment (markxnelson) [reply operator/src/main/resources/scripts/introspectDomain.py]: not a biggie

Comment (tbarnes-us) [reply operator/src/main/resources/scripts/introspectDomain.py]: Maybe ignore this comment.  It'd be painful to replace every mbean get in this script with a 'WA(' variant, so maybe we can assume the 'WA' issue only applies to certain mbeans (log, ssl, and dynamic)?   

Any thoughts Johnny? @jshum2479

Comment (rjeberhard) [reply operator/src/main/resources/scripts/introspectDomain.py]: I like the "OrNone" option

**Pull request merged**